whiteboard/index.js

```diff
@@ -0,0 +1,23 @@
+const fastify = require('fastify')({
+    logger: true,
+})
+const path = require('path')
+
+fastify.register(require('fastify-static'), {
+    root: path.join(__dirname, 'public'),
+    prefix: '/'
+})
+
+fastify.register(require('fastify-websocket'))
+
+fastify.get('/ws', { websocket: true }, (connection, req) => {
+  connection.socket.on('message', message => {
+    fastify.websocketServer.clients.forEach((client) => {
+      if (client.readyState === 1 && client !== connection.socket) {
+        client.send(message)
+      }
+    })
+  })
+})
+
+fastify.listen(process.env.PORT || 8080)
```

Comment:
```suggestion
fastify.listen(process.env.PORT || 3000)
```

Reply:
also note that you should specify the address as the second argument to .listen. see how the other examples do it